docs/tutorials/send_receive_ingress.md

```diff
@@ -96,41 +195,47 @@ spec:
     - addressConfigurations."APP.JOBS".queueConfigs."APP.JOBS".routingType=ANYCAST
     - addressConfigurations."APP.COMMANDS".routingTypes=MULTICAST
 EOF
+
+activemqartemis.broker.amq.io/send-receive created
 ```
 
 Wait for the Broker to be ready:
 
-```console
-kubectl get ActivemqArtemis  --watch
-NAME           READY   AGE
-send-receive   False   14s
-send-receive   True    51s
-send-receive   True    51s
+```{"stage":"deploy"}
+$ kubectl wait ActiveMQArtemis send-receive --for=condition=Ready --namespace=send-receive-project --timeout=240s
+
+activemqartemis.broker.amq.io/send-receive condition met
 ```
 
 #### Create a route to access the ingress:
 
 Check for the ingress availability:
 
-```console
+```
 $ kubectl get ingress --show-labels
-NAME                                 CLASS   HOSTS                                                                              ADDRESS         PORTS     AGE   LABELS
-send-receive-sslacceptor-0-svc-ing   nginx   send-receive-sslacceptor-0-svc-ing-send-receive-project.localresolvconf.com   192.168.39.57   80, 443   18h   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
+NAME                                 CLASS   HOSTS                                                                     ADDRESS         PORTS     AGE   LABELS
+send-receive-sslacceptor-0-svc-ing   nginx   ing.sslacceptor.send-receive-0.send-receive-project.demo.artemiscloud.io  192.168.39.68   80, 443   18s   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
 ```
 
-> [!NOTE]
-> The ingress hostname is resolving to
-> `send-receive-sslacceptor-0-svc-ing-send-receive-project.localresolvconf.com`.
-> This is because we have specified in the broker `spec` an `ingressDomain`.
-> This field must be populated for the operation to succeed.
+#### Populate /etc/hosts
+
+Get the url for the ingress
+
+```{"stage":"etc", "variables":["INGRESS_URL"], "runtime":"bash", "label":"get the ingress host"}
+$ INGRESS_URL=$(kubectl get ingress send-receive-sslacceptor-0-svc-ing -o json | jq -r '.spec.rules[] | .host')
+```
+
+Get minikube's ip
+
+```{"stage":"etc", "variables":["CLUSTER_IP"], "runtime":"bash", "label":"get the cluster ip"}
+$ CLUSTER_IP=$(minikube ip)
+```
+Populate the `/etc/hosts` file with an entry for the ingress url:
 
-For instance, you can add a new entry in your `/etc/hosts` to make this domain
-point to the associated address, in our example: `192.168.39.57`
+```{"stage":"etc", "env":["INGRESS_URL", "CLUSTER_IP"], "runtime":"bash", "label":"populate /etc/hosts", "id":"hosts_addition"}
+$ sudo echo "$CLUSTER_IP  $INGRESS_URL" | sudo tee -a /etc/hosts
```

Comment:
That looks super useful.

Quick thought, to use it, that would mean that the `dns-name` of the cluster should be `${CLUSTER_IP}.nip.io`. But since we can't know the ip address of the cluster before starting it (even though that's often `192.168.1.2`) aren't stuck with a chicken and egg problem?

Unless I misunderstood the `dns-domain` of the minikube instance command line.

Reply:
We are using nip.io now, thanks for the suggestion